src/controllers/authController.ts

```diff
@@ -0,0 +1,55 @@
+import { type Context } from 'hono';
+import { type UserContext } from '../models/contextModel';
+import { authService } from '../services/authService';
+import { deleteCookie, getCookie, setCookie } from 'hono/cookie';
+
+const cookieMaxAge = 15 * 24 * 60 * 60 * 1000; // 15 days in ms
+
+export const authController = {
+  async signUp(c: UserContext) {
```

Comment:
signUp should simply create a user and return something without tokens. Tokens are necessary for authentication(checking is the user is logged in)